src/balance.rs

```diff
@@ -155,7 +160,7 @@ where
 
     /// Does a state checking on initial state of cheques with current state.
     fn cmp_with_initial_state(&self) -> DispatchResult {
-        let cheques: Ref<HashMap<AccountIdOf<T>, BalanceOf<T>>> = self.cheques.borrow();
+        let cheques: Ref<HashMap<Vec<u8>, BalanceOf<T>>> = self.cheques.borrow();
```

Comment:
We could use an alias here to give the better context
`type EncodedAccount = Vec<u8>;`

```suggestion
        let cheques: Ref<HashMap<EncodedAccount>, BalanceOf<T>>> = self.cheques.borrow();
```

Reply:
Love it :+1:
Done